ui/js/actions/app.js

```diff
@@ -219,3 +219,15 @@ export function doChangeVolume(volume) {
     });
   };
 }
+
+export function doOpenOverlayMedia() {
+  return {
+    type: types.OPEN_OVERLAY_MEDIA,
+  };
+}
+
+export function doCloseOverlayMedia() {
```

Comment:
I also suspect this action can be avoided, and instead just be `doSetPlayingUri(null)`.